CosmosTrigger/function.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Can we name this function after a domain operation? For example `OnServiceChange`.

Reply:
Fix with 1cc968d